js/setup.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var userDialog = document.querySelector('.setup');
+userDialog.classList.remove('hidden');
+
+var WIZARD_NAME = ['Иван', 'Хуан', 'Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var WIZARD_SUBNAME = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var WIZARD_COATCOLOR = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var WIZARD_EYESCOLOR = ['black', 'red', 'blue', 'yellow', 'green'];
+
+
+var perem = function arrayRandElement(arr) {
+  var rand = Math.floor(Math.random() * arr.length);
+  var prm = arr[rand];
+  return prm;
+};
+
+var wizards = [];
+
+for (var i = 0; i < 4; i++) {
+  var wizard = {
+    name: perem(WIZARD_NAME) + ' ' + perem(WIZARD_SUBNAME),
+    coatColor: perem(WIZARD_COATCOLOR),
+    eyesColor: perem(WIZARD_EYESCOLOR)
+  }
+ ;
+  wizards [i] = wizard;
+}
+
+var similarListElement = userDialog.querySelector('.setup-similar-list');
+
+var similarWizardTemplate = document.querySelector('#similar-wizard-template')
```

Comment:
```suggestion
var similarWizardTemplateElement = document.querySelector('#similar-wizard-template')
```

Все переменные в которые выбираем элемент должна иметь окончание `Element`